lib/piwik/actions.rb

```diff
@@ -58,6 +58,14 @@ def outlinks params = {}
       def outlink(outlink_url, params = {})
         getOutlink(defaults.merge(params).merge(:outlinkUrl => outlink_url))
       end
+
+      def sitesearch_keywords params = {}
+        getSiteSearchKeywords(defaults.merge(params))
+      end
+
+      def sitesearch_no_result_keywords params = {}
+        getSiteSearchNoResultKeywords(defaults.merge(params))
+      end
```

Comment:
downcasing the original method names gives

```suggestion
      def site_search_keywords params = {}
        getSiteSearchKeywords(defaults.merge(params))
      end

      def site_search_no_result_keywords params = {}
        getSiteSearchNoResultKeywords(defaults.merge(params))
      end
```